src/config.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import hathorLib from '@hathor/wallet-lib';
+
+/**
+ * This file contains all variables a user has to modify if he wants to create
+ * an app for his own token. Files should never import them directly (they are
+ * prepended with '_'). Instead, constants.js imports and re-exports them.
+ */
```

Comment:
@pedroferreira1 added the variables here. I kept `INITIAL_TOKENS` and `LOCK_TIMEOUT` on the other file. What do you think?

Reply:
Good!